src/components/Auth/SignUp/SignUp.react.js

```diff
@@ -405,7 +432,26 @@ export default class SignUp extends Component {
                 textFieldStyle={{ padding: '0px' }}
               />
             </div>
-
+            <div
+              style={{
+                marginTop: '10px',
+              }}
+            >
+              <Recaptcha
+                sitekey="6LfPZGAUAAAAANY4ZLR_lSat_YCuqqcV3sl84ulW"
```

Comment:
To be able to maintain this we should have such kind of keys in a configuration file.

Reply:
I thought of this before, how and where should I put the key? @Orbiter

Reply:
Add a config file into the root directory of the repo and make Recaptcha use the key from the config file.